q2_longitudinal/plugin_setup.py

```diff
@@ -189,6 +191,7 @@
         'ci': 'Size of the confidence interval to plot on control chart.',
         'plot_control_limits': ('Plot global mean and control limits (2X and '
                                 '3X standard deviations).'),
+        'xtick_interval': 'Interval between major tick marks on x axis.'
```

Comment:
Can you add a note here to indicate that some autoscaling happens? I want to make it clear to a user that things might come out in the viz slightly different than expected.